src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,209 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Prints the relevant messages.
+ */
+public class Ui {
+    /**
+     * Prints logo "Welcome" greeting.
+     */
+    public void logoMsg() {
+        String logo = " ___       __   _______   ___       ________  "
+                + "________  _____ ______   _______      \n"
+                + "|\\  \\     |\\  \\|\\  ___ \\ |\\  \\     "
+                + "|\\   ____\\|\\   __  \\|\\   _ \\  _   \\|\\  ___ \\     \n"
+                + "\\ \\  \\    \\ \\  \\ \\   __/|\\ \\  \\    \\ \\  \\___|"
+                + "\\ \\  \\|\\  \\ \\  \\\\\\__\\ \\  \\ \\   __/|    \n"
+                + " \\ \\  \\  __\\ \\  \\ \\  \\_|/_\\ \\  \\    \\ \\  \\    "
+                + "\\ \\  \\\\\\  \\ \\  \\\\|__| \\  \\ \\  \\_|/__  \n"
+                + "  \\ \\  \\|\\__\\_\\  \\ \\  \\_|\\ \\ \\  \\____\\ \\  \\____"
+                + "\\ \\  \\\\\\  \\ \\  \\    \\ \\  \\ \\  \\_|\\ \\ \n"
+                + "   \\ \\____________\\ \\_______\\ \\_______\\ \\_______"
+                + "\\ \\_______\\ \\__\\    \\ \\__\\ \\_______\\\n"
+                + "    \\|____________|\\|_______|\\|_______|\\|_______|\\|_______"
+                + "|\\|__|     \\|__|\\|_______|"
+                + "\n\n WELCOME to DukeBT!";
+        System.out.println(logo);
+    }
+
+    /**
+     * Prints greeting.
+     */
+    public void greetingMsg() {
+        String greetingMsg = "\n___________________________________________________________"
+                + "\n |°‿°|"
+                + "\n Hello! I'm DukeBT."
+                + "\n What can I do for you?"
+                + "\n\n **Type 'help' to see what I can do. |^_^|"
+                + "\n___________________________________________________________\n";
+        System.out.println(greetingMsg);
+    }
```

Comment:
Would be good to abstract out the `"_____"` as an echo function, so that it can be changed easily in the future. Good use of line breaks for readability

Reply:
Good point! Will amend it accordingly, thank you!